src/Akeneo/Bundle/ElasticSearchBundle/AkeneoElasticSearchBundle.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Akeneo\Bundle\ElasticSearchBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+/**
+ * AkeneoElasticSearchBundle is a simple and stupid wrapper for the PHP ElasticSearch library.
+ * To learn more see {@link https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/index.html}
+ *
+ * @author    Julien Janvier <j.janvier@gmail.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/MIT MIT
```

Comment:
* @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)

I would say your IDE is wrongly configured.